sync/sync.go

```diff
@@ -2,11 +2,10 @@ package sync
 
 import (
 	"fmt"
-	"os"
-
 	"github.com/please-build/buildtools/build"
 	"github.com/please-build/buildtools/labels"
 	"golang.org/x/mod/modfile"
+	"os"
```

Comment:
I don't think this order is correct it should be 
```
builtin

3rd party

1st party
```

Reply:
True! Not sure why my IDE decided to scramble that. Fixed